components/common/src/error.rs

```diff
@@ -34,6 +34,7 @@ pub enum Error {
     DepotClient(depot_client::Error),
     FileNameError,
     HabitatCore(hcore::Error),
+    InsufficientPermissions,
```

Comment:
We might consider naming this error something a little less generic. We need a user with elevated privileges in the form of sudo or the root user. I'd update the description to include that as well. A dev may come along later and re-use this error because they're converting a `std::io::Error` into something more contextual and

![gif-keyboard-1799408427863706874](https://cloud.githubusercontent.com/assets/54036/19090202/3accada6-8a32-11e6-84b5-69e2743690b1.gif)